database/procedures/process_checkout.sql

```diff
@@ -60,15 +63,15 @@ BEGIN
     -- ****************************************
     --
     -- Check if we have sufficient inventory for all items
-    -- BEGIN TRY
-    --     EXEC CheckInsufficientInventory @CartItems;
-    -- END TRY
-    -- BEGIN CATCH
-    --     SELECT 
-    --         'Error' AS Status,
-    --         ERROR_MESSAGE() AS message;
-    --     RETURN;
-    -- END CATCH
```

Comment:
Misty, we need to leave the CheckInsufficientInventory commented out. See comments above.